<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorSun</class>
 <widget class="QWidget" name="InspectorSun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sun</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Shadow Refresh</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QComboBox" name="m_pShadowRefreshCB">
        <item>
         <property name="text">
          <string>Static</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dynamic</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Color Mode:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="m_pColorModeCB">
        <item>
         <property name="text">
          <string>Color</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Temperature</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Intensity</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="m_pIntensityEdit">
          <property name="text">
           <string>1200</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Lux</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="m_pTemperatureEdit">
          <property name="text">
           <string>5300</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Kelvin</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pDirectionXEdit">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pDirectionYEdit">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pDirectionZEdit">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Direction</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="m_pPickColorButton">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pPickColorButton2">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Pick</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pColorModeCB</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InspectorSun</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pPickColorButton</sender>
   <signal>clicked()</signal>
   <receiver>InspectorSun</receiver>
   <slot>pickColorFromDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pPickColorButton2</sender>
   <signal>clicked()</signal>
   <receiver>InspectorSun</receiver>
   <slot>pickColorFromDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pTemperatureEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorSun</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pIntensityEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorSun</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pDirectionXEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorSun</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pDirectionYEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorSun</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pDirectionZEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorSun</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pShadowRefreshCB</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InspectorSun</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valueChanged()</slot>
  <slot>pickColorFromDialog()</slot>
 </slots>
</ui>
